test/integration/auth_test.go

```diff
@@ -844,14 +779,8 @@ func (a *trackingAuthorizer) Authorize(attributes authorizer.Attributes) error {
 func TestAuthorizationAttributeDetermination(t *testing.T) {
 	framework.DeleteAllEtcdKeys()
 
-	etcdStorage, err := framework.NewEtcdStorage()
-	if err != nil {
-		t.Fatalf("unexpected error: %v", err)
-	}
-	expEtcdStorage, err := framework.NewExtensionsEtcdStorage(nil)
-	if err != nil {
-		t.Fatalf("unexpected error: %v", err)
-	}
+	etcdStorage := framework.NewEtcdStorage()
```

Comment:
It seems that this one (and the following lines, e.g. the whole storageDestinations-related code) is no longer needed.
Can you please remove it?